Editor/BootstrapperConfig.cs

```diff
@@ -16,6 +16,11 @@ static class BootstrapperConfig
 
         public const string GitHubRepositoryURL = @"ssh://git@github.com:StansAssets/OneSignal-Unity-SDK.git";
```

Comment:
Just noting here so we don't forget. Replace `StansAssets` with `OneSignal` here before releasing.

Reply:
This has to be done in this PR actually.  Thx for heads up.